exercises/ruby-1-b/ruby_1_b_example.rb

```diff
@@ -1,12 +1,95 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
-  end
-  word_vowels.sort_by {|w,c| -c}.first.first
+class Library
```

Comment:
Nice example. Could you try to create three different examples that all use different approaches? Similar to what `ruby-1-a` did? https://github.com/exercism/research_experiment_1/tree/master/exercises/ruby-1-a

Reply:
I'll create 2 more examples once I'm sure which stuff should be moved to the second part 👍